golem-component-service/db/migration/postgres/002__add_component_type.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE component_versions
+    ADD COLUMN IF NOT EXISTS component_type integer NOT NULL DEFAULT 0;
```

Comment:
can `component_type` change with version? if not this column should be added to `components` table

Reply:
ok, i see now that there is component type in update, so it can be updated, then it is ok ...

Reply:
it can